frb/dm_kde/transient_data/frbcat.csv

```diff
@@ -0,0 +1,111 @@
+﻿"frb_name","utc","telescope","rop_raj","rop_decj","rop_gl","rop_gb","rmp_dm","rmp_width","rmp_snr"
```

Comment:
Can you move this file into frb/data/FRBs ?

And rename it like the other ones in there.
Thanks!

Reply:
Done :)